pkg/networkservice/common/excludedprefixes/server.go

```diff
@@ -37,10 +37,11 @@ import (
 )
 
 type excludedPrefixesServer struct {
-	ctx        context.Context
-	prefixPool atomic.Value
-	once       sync.Once
-	configPath string
+	ctx              context.Context
+	prefixPool       atomic.Value
+	once             sync.Once
+	configPath       string
+	previousPrefixes map[string]struct{}
```

Comment:
Could you explain why is this not per connection?

Reply:
ok, fixed